lib/grape/validations/validators/coerce.rb

```diff
@@ -5,6 +5,11 @@ class API
 
   module Validations
     class CoerceValidator < Base
+      def initialize(*_args)
+        super
+        @converter = Types.build_coercer(type, @option[:method])
```

Comment:
This change is to make sure `@converter` is only created once, rather than in each validator dup.

`coerce_spec.rb` is expecting this behavior.